Formula/jackett.rb

```diff
@@ -0,0 +1,60 @@
+class Jackett < Formula
+  desc "API Support for your favorite torrent trackers"
+  homepage "https://github.com/Jackett/Jackett"
+  url "https://github.com/Jackett/Jackett/archive/refs/tags/v0.20.805.tar.gz"
+  sha256 "5b37a01517d2230879fdd0b5f5a3344abcc7718adee9584c2b7b40e5b99b721b"
+  license "GPL-2.0-only"
+  head "https://github.com/Jackett/Jackett.git", branch: "master"
+
+  depends_on "dotnet"
+
+  def install
+    cd "src" do
+      os = OS.mac? ? "osx" : OS.kernel_name.downcase
+      arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+
+      args = %W[
+        --configuration Release
+        --framework net#{Formula["dotnet"].version.major_minor}
+        --output #{libexec}
+        --runtime #{os}-#{arch}
+        --no-self-contained
+        /p:AssemblyVersion=#{version}
+        /p:FileVersion=#{version}
+        /p:InformationalVersion=#{version}
+        /p:Version=#{version}
+      ]
+
+      system "dotnet", "publish", "Jackett.Server", *args
+    end
+
+    (bin/"jackett").write_env_script libexec/"jackett", "--NoUpdates",
+      DOTNET_ROOT: "${DOTNET_ROOT:-#{Formula["dotnet"].opt_libexec}}"
+  end
+
+  service do
+    run opt_bin/"jackett"
+    keep_alive true
+    working_dir libexec
+    log_path var/"log/jackett.log"
+    error_log_path var/"log/jackett.log"
+  end
+
+  test do
+    assert_match(/^Jackett v#{Regexp.escape(version)}$/, shell_output("#{bin}/jackett --version 2>&1; true"))
+
+    port = free_port
+
+    pid = fork do
+      exec "#{bin}/jackett", "-d", testpath, "-p", port.to_s
+    end
+    sleep 3
```

Comment:
Let's try this to see if it fixes it
```suggestion
    sleep 10
```

Reply:
Sure. Thanks.